docs/hardhat-network/README.md

```diff
@@ -269,162 +147,18 @@ eth_call
 
 This logging is enabled by default when using Hardhat Network's node (i.e. `npx hardhat node`), but disabled when using the in-process Hardhat Network provider. See [Hardhat Network's config](../config/README.md#hardhat-network) to learn more about how to control its logging.
 
-## The `debug_traceTransaction` method
+### The `debug_traceTransaction` method
 
-You can get debug traces of already mined transactions using the `debug_traceTransaction` RPC method. The returned object has a detailed description of the transaction execution, including a list of steps describing each executed opcode and the state of the EVM at that point.
+You can get debug traces of already-mined transactions using the `debug_traceTransaction` RPC method. The returned object has a detailed description of the transaction execution, including a list of steps describing each executed opcode and the state of the EVM at that point.
 
-To get a trace, call this method with the hash of the transaction as its argument:
+If you are using [mainnet forking](../guides/mainnet-forking.html) with an archive node, you can get traces of transactions from the remote network even if the node you are using doesn't support `debug_traceTransaction`.
 
-```js
-const trace = await hre.network.provider.send("debug_traceTransaction", [
-  "0x123...",
-]);
-```
+For more details, see [the reference documentation for this method](./reference/README.md#debug-tracetransaction).
 
-You can also selectively disable some properties in the list of steps:
-
-```js
-const trace = await hre.network.provider.send("debug_traceTransaction", [
-  "0x123...",
-  {
-    disableMemory: true,
-    disableStack: true,
-    disableStorage: true,
-  },
-]);
-```
+## Why _wouldn't_ I want to use it?
```

Comment:
I'm not sure if I agree with this subtitle. IMO the limitation section was mostly to help users understand what can go wrong and how to deal with those (e.g. disable the optimizer).

Maybe this should be moved to reference?

Reply:
What I actually did was move the details, of what used to be here in this doc, over to the Reference, and left a sort of summary explanation here.  Given your comment here, I just deleted this section: https://github.com/nomiclabs/hardhat/pull/1657/commits/6e4cd0711c19fd62d2d00dd5304684627dfa20ec